docs/advanced_topics/images/feature_detection.md

```diff
@@ -8,7 +8,7 @@ Feature detection uses third-party tools to detect faces/features in an image wh
 
 ## Installation
 
-Two third-party tools are known to work with Wagtail: One based on [OpenCV](https://opencv.org/) for general feature detection and one based on [Rustface](https://github.com/torchbox/rustface-py/) for face detection.
+Two third-party tools are known to work with Wagtail for feature detection: One based on [OpenCV](https://opencv.org/) for general feature detection and one based on [Rustface](https://github.com/torchbox/rustface-py/) for face detection.
```

Comment:
This whole page is about this topic, and we already specify what the tools are for exactly a few words further. This just seems like unnecessary repetition.

```suggestion
Two third-party tools are known to work with Wagtail: One based on [OpenCV](https://opencv.org/) for general feature detection and one based on [Rustface](https://github.com/torchbox/rustface-py/) for face detection.
```